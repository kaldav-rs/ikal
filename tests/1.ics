BEGIN:VCALENDAR
PRODID:-//Nextcloud calendar v1.5.0
VERSION:2.0
CALSCALE:GREGORIAN
BEGIN:VEVENT
CREATED:20170209T192358
DTSTAMP:20170209T192358
LAST-MODIFIED:20170209T192358
UID:5UILHLI7RI6K2IDRAQX7O
SUMMARY:Vers
CLASS:PUBLIC
STATUS:CONFIRMED
DTSTART;VALUE=DATE-TIME;TZID=Europe/Paris:20150219T190000
DTEND;VALUE=DATE:20170210
END:VEVENT
END:VCALENDAR
