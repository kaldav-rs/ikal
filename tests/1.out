Done(
    "\r\n",
    Ok(
        VCalendar {
            prodid: "-//Nextcloud calendar v1.5.0",
            version: "2.0",
            calscale: Some(
                "GREGORIAN"
            ),
            event: VEvent {
                created: 2017-02-09T19:23:58+01:00,
                dtstamp: 2017-02-09T19:23:58+01:00,
                last_modified: 2017-02-09T19:23:58+01:00,
                uid: "5UILHLI7RI6K2IDRAQX7O",
                summary: "Vers",
                class: Public,
                status: Confirmed,
                dt_start: 2015-02-19T19:00:00+01:00,
                dt_end: 2017-02-10T00:00:00+01:00,
                extra: {}
            }
        }
    )
)
